spec.html

```diff
@@ -15939,7 +15939,7 @@ <h1>
         1. Let _size_ be the length of _string_.
         1. Assert: _position_ &ge; 0 and _position_ &lt; _size_.
         1. Let _first_ be the code unit at index _position_ within _string_.
-        1. Let _cp_ be the code point whose numeric value is that of _first_.
+        1. Let _cp_ be the code point whose numeric value is _first_.
```

Comment:
The intent is to compare two numeric values: the numeric value of `_cp_` and the numeric value of `_first_`. You're proposing to compare the numeric value of `_cp_` with `_first_`, which is a code unit.

If you don't like the "is that of" construction, you could expand it to:
```suggestion
        1. Let _cp_ be the code point whose numeric value is the numeric value of _first_.
```

And similarly for the other cases.

Reply:
Ah, yes, I didn't check these and assumed they were all comparing against numeric values. I will accept your suggestion.